src/m_menu.c

```diff
@@ -3343,8 +3347,8 @@ setup_menu_t stat_settings2[] =
   {"CROSSHAIR",S_SKIP|S_TITLE,m_null,M_X,M_Y+stat2_title3*M_SPC },
 
   {"ENABLE CROSSHAIR",      S_CHOICE,m_null,M_X,M_Y+stat2_xhair*M_SPC,       {"hud_crosshair"}, 0, M_UpdateCrosshairItems, crosshair_str},
-  {"COLOR BY PLAYER HEALTH",S_YESNO, m_null,M_X,M_Y+stat2_xhairhealth*M_SPC, {"hud_crosshair_health"}},
-  {"HIGHLIGHT ON TARGET",   S_YESNO, m_null,M_X,M_Y+stat2_xhairtarget*M_SPC, {"hud_crosshair_target"}, 0, M_UpdateCrosshairItems},
+  {"COLOR BY PLAYER HEALTH",S_YESNO ,m_null,M_X,M_Y+stat2_xhairhealth*M_SPC, {"hud_crosshair_health"}},
+  {"COLOR BY TARGET",       S_CHOICE,m_null,M_X,M_Y+stat2_xhairtarget*M_SPC, {"hud_crosshair_target"}, 0, M_UpdateCrosshairItems, crosshair_target_str},
```

Comment:
"COLOR ON TARGET"?

Reply:
Sorry, is that a question or a suggestion?

Reply:
A suggestion. I've left the question mark, because I'm not a native speaker myself. But, I still think it makes more sense like this. 😉

Reply:
Well, I don't think neither "Color By Target Highlight" nor "Color On Target Highlight" make sense, but "Color By Target Health" does sound more natural than "Color On Target Health", to me at least. Perhaps we could get feedback from other users? Regardless, I'll be awaiting for your confirmation.

Reply:
if "color by player health" is ok, I would assume "color by target health" is ok too 

oh and BTW, another PR? oh my!
what's going to be next? shaded crosshair? ;)

Reply:
I'm reading it more like "Color On Target: Highlight" and "Color On Target: Health", but I don't have a *strong* opinion on this (that's why I added that question mark in the first place). Let's hear what others think, but let's not get kept back by this.

Reply:
> what's going to be next? shaded crosshair? ;)

I don't think so. Personally, I'l like a translucent crosshair, but we'd first need a translucent and color translated patch drawing function for this. After that, I think we are done with crosshair options.